Integrations/integration-RTIR.yml

```diff
@@ -585,16 +684,54 @@ script:
             demisto.setLastRun({'ticket_id': max_ticket_id})
         demisto.incidents(incidents)
 
+
     ''' EXECUTION CODE '''
```

Comment:
the `http_request`, `login` and `logout` should not be in the execution section, but in helper functions section